src/main/java/com/example/TwoRoomsBoom/models/Lobby.java

```diff
@@ -17,9 +21,85 @@
 @NoArgsConstructor
 public class Lobby {
     private String lobbyId;
+    private Game game;
     private List<Role> rolesAdded;
     private List<Player> players;
     private Set<Player> readyPlayers;
     private Player host;
-    private String settings;
+    @Default
+    private boolean enableBury = false;
+
+    public void startGame() {
+        this.game = new Game();
+    }
+
+    public void addPlayers(Player player) {
+        if (!this.players.contains(player)) {
+            this.players.add(player);
+        }
+    }
+
+    public void kickPlayer(Player player) {
+        if (this.players.contains(player)) {
+            this.players.remove(player);
+        }
+    }
+
+    public void addRole(Role role) {
+        // Can have multiple Blue/Red team role in play but other roles cannot
+        if (role.getName().equals("Blue Team") || role.getName().equals("Red Team")) {
+            // Should be equal amount of Blue/Red team added
+            if (role.getName().equals("Blue Team")) {
+                this.rolesAdded.add(new RedTeam());
+            } else {
+                this.rolesAdded.add(new BlueTeam());
+            }
+            this.rolesAdded.add(role);
+        } else if (!this.rolesAdded.contains(role)) {
+            this.rolesAdded.add(role);
+        }
+    }
+
+    public void removeRole(Role role) {
+        if (this.rolesAdded.size() > 0 && this.rolesAdded.contains(role)) {
+            // Removes both Blue/Red Team
+            if (role.getName().equals("Blue Team") || role.getName().equals("Red Team")) {
+                Iterator<Role> iter = rolesAdded.iterator();
+                while (iter.hasNext()) {
```

Comment:
Yea I think if we were to add additional roles this would be a long operation especially if there's a lot of adding and removing of roles. lowkey think that the roles should be stored differently